tests/results/reconfig/v1.3.0/1.3.0-oss.md

```diff
@@ -0,0 +1,101 @@
+# Results
+
+## Test environment
+
+NGINX Plus: false
+
+GKE Cluster:
+
+- Node count: 3
+- k8s version: v1.29.6-gke.1038001
+- vCPUs per node: 2
+- RAM per node: 4019168Ki
+- Max pods per node: 110
+- Zone: us-central1-c
+- Instance Type: e2-medium
+
+## Test 1 NumResources 30
+
+### Reloads and Time to Ready
+
+- TimeToReadyTotal: 1
```

Comment:
Results are different, in all the areas they differ they differ by these results being faster / less reloads / less batches.

I think this could be because before we had a manual script deploying resources while now some is running through an `exec` command while others are running through the k8s client which would be faster.

Reply:
This has been a common trend as we've been automating the tests. It's going to be hard to compare the results from the automation to the old manual tests because of the differences in timing.

What's important is that the automated tests are consistent when run multiple times.